### PR TITLE
Dev brokenlinks

### PR DESCRIPTION
**Pull request by doerners**
Some of the broken links that were identified during the developer workshop are replaced by non-broken equivalents. For others there might be some need for discussion, i.e. the situation isn't clear as to whether to delete certain broken links entirely or if there even is a working alternative. Therefore not all of the possible changes are encompassed by the pull request.

Also some older http addresses were changed to https.

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ MEI and MusicXML share some similarities -- both of them encode music notation (
 
 ## What is the history of MEI?
 
-In 1999, Perry Roland created an XML schema (DTD) for the representation of music notation. Eventually, this DTD became known as MEI, drawing on the same principles that guided the creation of the [Text Encoding Initiative](http://www.tei-c.org/) (TEI). Perry presented his initial work at the first [International Society for Music Information Retrieval](http://ismir.net/conferences.html) in 2000.
+In 1999, Perry Roland created an XML schema (DTD) for the representation of music notation. Eventually, this DTD became known as MEI, drawing on the same principles that guided the creation of the [Text Encoding Initiative](http://www.tei-c.org/) (TEI). Perry presented his initial work at the first [International Society for Music Information Retrieval](https://ismir.net/conferences/) in 2000.
```
Comment (musicEnfanthen) [inline about/index.md:44]:
```suggestion
In 1999, Perry Roland created an XML schema (DTD) for the representation of music notation. Eventually, this DTD became known as MEI, drawing on the same principles that guided the creation of the [Text Encoding Initiative](http://www.tei-c.org/) (TEI). Perry presented [his initial work](http://ismir2000.ismir.net/papers/roland_paper.pdf) at the first [International Society for Music Information Retrieval](https://ismir2000.ismir.net/) in 2000.
```

File: _projects/syrinx.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ contributors:
     - "Raffaele Viglianti, London"
 archive: true
 ---
-_Syrinx_ (_La Flûte de Pan_) by Claude Debussy (1862 - 1918) is a short piece for flute solo originally composed as a theatrical interlude called La Flûte de Pan. Comparing the first edition (published posthumously in 1927), a manuscript discovered in 1991, a 1996 edition, and a 1922 _Music and Letters_ article by the first performer of _La Flûte de Pan_, [this report](http://legacy.cch.kcl.ac.uk/syrinx/about.html) investigates the potential of digital representation and presentation of comparative editions of music using MEI.
\ No newline at end of file
+_Syrinx_ (_La Flûte de Pan_) by Claude Debussy (1862 - 1918) is a short piece for flute solo originally composed as a theatrical interlude called La Flûte de Pan. Comparing the first edition (published posthumously in 1927), a manuscript discovered in 1991, a 1996 edition, and a 1922 _Music and Letters_ article by the first performer of _La Flûte de Pan_, [this report](https://core.ac.uk/download/pdf/147825165.pdf) investigates the potential of digital representation and presentation of comparative editions of music using MEI.
```
Comment (musicEnfanthen) [inline _projects/syrinx.md:10]:
```suggestion
_Syrinx_ (_La Flûte de Pan_) by Claude Debussy (1862 - 1918) is a short piece for flute solo originally composed as a theatrical interlude called La Flûte de Pan. Comparing the first edition (published posthumously in 1927), a manuscript discovered in 1991, a 1996 edition, and a 1922 _Music and Letters_ article by the first performer of _La Flûte de Pan_, [this report](https://core.ac.uk/download/pdf/147825165.pdf#page=41) investigates the potential of digital representation and presentation of comparative editions of music using MEI.
```

Comment (musicEnfanthen) [reply _projects/syrinx.md:10]: However, I am not sure if that's the exact target that was addressed by the broken link: http://legacy.cch.kcl.ac.uk/syrinx/about.html

Maybe @raffazizzi can confirm?

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]
Thanks a lot, @doerners . Great work.

After merging this, we can implement the link checker action, so we can identify any other broken links.

File: about/index.md (hunk 1 above)
Comment (rettinghaus) [reply about/index.md:44]: The name of the ISMIR link should be "ISMIR conference".
@musicEnfanthen I wouldn't add a direct link to the paper here, as it is available in the bibliography.

**Review by rettinghaus — APPROVED** [review]

File: about/index.md (hunk 1 above)
Comment (musicEnfanthen) [reply about/index.md:44]: Ok, agreed

**Review by rettinghaus — APPROVED** [review]

File: _projects/syrinx.md (hunk 2 above)
Comment (rettinghaus) [reply _projects/syrinx.md]: The original address was [archived](https://web.archive.org/web/20100712140310/http://legacy.cch.kcl.ac.uk/syrinx/about.html), but the _Digital Edition Demo_ doesn't work anymore. The original report can be found here: http://dh2010.cch.kcl.ac.uk/academic-programme/abstracts/papers/html/ab-819.html 
There's also a PDF version available.

**Review by rettinghaus — COMMENTED** [review]
Looks good, thank you!

**Comment by rettinghaus** [comment]
@doerners do you want to make further changes, or should I merge this?

**Comment by doerners** [comment]
@rettinghaus well there are still some unresolved questions regarding the broken links in issue #189 
I'm not sure what's the best course of action now...any suggestions?

**Comment by rettinghaus** [comment]
@doerners I would say that broken links should always be fixed as soon as possible. And #189 will most likely take some time before being fully resolved.

**Comment by doerners** [comment]
@rettinghaus I second your point(s), so I would suggest that you do the merge now since there are no immediated changes planned on my side. That is of course if there are no objections from the community.

**Pull request merged**